zero/gkyl_comm_priv.h

```diff
@@ -95,6 +95,7 @@ struct gkyl_comm_priv {
   // FOLLOWING NEED A DECOMP 
 
   gkyl_array_allgather_t gkyl_array_allgather; // gather local arrays to global array
+  gkyl_array_allgather_t gkyl_array_allgather_host; // ?? gather local arrays to global array on host
```

Comment:
@Maxwell-Rosen what's up with the ??

Reply:
Not sure. I didn't put them there. @ammarhakim put these ??